### PR TITLE
Update demo instructions

### LINKED ISSUE #346: Instructions for hospitals

### LINKED ISSUE #347: Setup instructions for Benchmark owners

### LINKED ISSUE #381: [BUG] Udpate install and demonstration instructions
**Describe the bug**
The current state of the instructions is outdated. Instructions do not specify how to correctly launch and seed a local server, nor how to setup medperf for local execution. Dataset references are also outdated, and some of the requirements for running the demo are not specified inside the `requirements.txt`.

**To Reproduce**
Try to follow the demo instructions using the latest version of medperf

### PR DESCRIPTION
**Pull request by hasan7n**
Move the `how to run` section from the README, and create more documented tutorials on the documentation website.
closes #381. Closes #346. Closes #347.

### CODE REVIEW

**Comment by hasan7n** [comment]
TODO:
- update data hash for data owner tutorial and follow the tutorial myself

**Review by aristizabal95 — APPROVED** [review]
The documentation is looking crazy good! This is amazing Hasan, thanks for the hard work! I believe everything looks right and sufficiently detailed for anyone to be able to go along. My only comments would be:
- We should create a PR for removing references to our own forks on the code and documentation
- Let's ask Cassiano to take a deep look at the changes so that they're worded correctly, I noticed just a very few number of sentences that I would modify, but it would be best for someone with more experience on that to polish all the text.

I'll approve the PR. Feel free to decide if we want Cassiano to take a look at this before merging, or after merging on his own PR.

Thanks again!

File: examples/ChestXRay/xrv_densenet/mlcube/workspace/parameters_pc.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+model: "pc"
```
[the remark this answers is not in the document]
Comment (aristizabal95) [reply examples/ChestXRay/xrv_densenet/mlcube/workspace/parameters_pc.yaml:1]: Nevermind, I noticed the purpose later but I couldn't find the comment haha. Closing this

**Comment by hasan7n** [comment]
Thanks Alejandro! I agree to both points. I will merge this so that we have it hosted, will create an issue and assign it to Cassiano

**Pull request merged**